src/main/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravVurderingRequestDTO.kt

```diff
@@ -16,6 +16,55 @@ fun AktivitetskravVurderingRequestDTO.toAktivitetskravVurdering(
     status = this.status,
     createdBy = createdByIdent,
     beskrivelse = this.beskrivelse,
-    arsaker = this.arsaker,
+    arsaker = arsaker.map { it.toVurderingArsak(this.status) },
     frist = this.frist,
 )
+
+enum class Arsak {
```

Comment:
Denne enumen brukes litt rundt om kring i api, domene og database nå. Hadde det vært bedre om den lå i en egen fil i domain i stedet for i ResponseDTO-fila?

Reply:
Nå er den bare her i presentasjons/web laget. Kan representere `arsaker` som `List<String>` også. Jeg bare tenkte det er best at den feiler på deserialisering enn når den skal konverteres til `AktivitetskravVurdering` 😊